types/node/events.d.ts

```diff
@@ -9,31 +7,27 @@ declare module "events" {
     }
 
-    namespace internal {
-        function once(emitter: NodeEventTarget, event: string | symbol): Promise<any[]>;
-        function once(emitter: DOMEventTarget, event: string): Promise<any[]>;
-         class EventEmitter extends internal {
-            /** @deprecated since v4.0.0 */
-            static listenerCount(emitter: EventEmitter, event: string | symbol): number;
-            static defaultMaxListeners: number;
+    function once(emitter: NodeEventTarget, event: string | symbol): Promise<any[]>;
+    function once(emitter: DOMEventTarget, event: string): Promise<any[]>;
+    class EventEmitter implements NodeJS.EventEmitter {
+        /** @deprecated since v4.0.0 */
+        static listenerCount(emitter: EventEmitter, event: string | symbol): number;
+        static defaultMaxListeners: number;
 
-            addListener(event: string | symbol, listener: (...args: any[]) => void): this;
-            on(event: string | symbol, listener: (...args: any[]) => void): this;
-            once(event: string | symbol, listener: (...args: any[]) => void): this;
-            prependListener(event: string | symbol, listener: (...args: any[]) => void): this;
-            prependOnceListener(event: string | symbol, listener: (...args: any[]) => void): this;
-            removeListener(event: string | symbol, listener: (...args: any[]) => void): this;
-            off(event: string | symbol, listener: (...args: any[]) => void): this;
-            removeAllListeners(event?: string | symbol): this;
-            setMaxListeners(n: number): this;
-            getMaxListeners(): number;
-            listeners(event: string | symbol): Function[];
-            rawListeners(event: string | symbol): Function[];
-            emit(event: string | symbol, ...args: any[]): boolean;
-            eventNames(): Array<string | symbol>;
-            listenerCount(type: string | symbol): number;
-        }
+        addListener(event: string | symbol, listener: (...args: any[]) => void): this;
+        on(event: string | symbol, listener: (...args: any[]) => void): this;
+        once(event: string | symbol, listener: (...args: any[]) => void): this;
+        prependListener(event: string | symbol, listener: (...args: any[]) => void): this;
+        prependOnceListener(event: string | symbol, listener: (...args: any[]) => void): this;
+        removeListener(event: string | symbol, listener: (...args: any[]) => void): this;
+        off(event: string | symbol, listener: (...args: any[]) => void): this;
+        removeAllListeners(event?: string | symbol): this;
+        setMaxListeners(n: number): this;
+        getMaxListeners(): number;
+        listeners(event: string | symbol): Function[];
+        rawListeners(event: string | symbol): Function[];
+        emit(event: string | symbol, ...args: any[]): boolean;
+        eventNames(): Array<string | symbol>;
+        listenerCount(type: string | symbol): number;
     }
-
-    export = internal;
```

Comment:
`EventEmitter` constructor is exported like this (and as property `EventEmitter`, see https://github.com/nodejs/node/blob/64eacd8741f23cb91569faf7ec793c83badb04e3/lib/events.js#L46

Reply:
Closing in favor of other discussion.